src/demo/routes/routes.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { api, SdkError, SdkErrors } from "@tonomy/tonomy-id-sdk";
+import useErrorStore from "../../common/stores/errorStore";
+import PrivateRoute from "./guards/PrivateRoute";
+import LogoutProvider from "../providers/LogoutProvider";
+import HomeScreen from "../pages/Home";
+import Callback from "../pages/Callback";
+import MainLayout from "../layout/MainLayout";
+import mainRoutes from "./mainRoutes";
+
+export default function RootRoutes(): JSX.Element {
+  const [isAuthenticated, setIsAuthenticated] = useState(false);
+  const errorStore = useErrorStore();
+
+  useEffect(() => {
+    const fetchAuthentication = async (): Promise<void> => {
+      try {
+        const user = await api.ExternalUser.getUser({ autoLogout: false });
```

Comment:
looks like you have the same login in two places. It is also here in the LogoutProvider.tsx
https://github.com/Tonomy-Foundation/Tonomy-App-Websites/pull/101/files#diff-4783907d845fc383c6872ba940b165d47cb9c4c30c3d74b8e371a08beafb7a46R23

This should only be in one place